manifest.xml

```diff
@@ -128,13 +128,15 @@
       <skill>intermediate</skill>
     </tutorial>
 
-    <tutorial title='Convert SDF to USD' ref='convert_sdf_to_usd' type='documentation'>
-      <markdown version='12'>convert_sdf_to_usd/tutorial.md</markdown>
+    <tutorial title='Interoperability with other modeling formats' ref='interoperability_formats' type='documentation'>
+      <markdown version='12'>interoperability_formats/tutorial.md</markdown>
```

Comment:
MJCF is only available with version 13 and later. Maybe we should mention that in the tutorial.